org/pr/installable-build.ts

```diff
@@ -131,10 +133,11 @@ async function getDownloadCommentText(status) {
 }
 
 export default async (status: Status) => {
+  var pippo = INSTALLABLE_BUILD_CONTEXTS.filter(s => status.context.match(s));
     if (status.state == "pending" && HOLD_CONTEXTS.includes(status.context)) {
```

Comment:
Seems like leftover from debugging session there 🙃

(TIL about the `debugger` command, seems useful!)

Reply:
Good catch! 🤦‍♂️ Clearly trying to do too many things in parallel.